project/build.properties

```diff
@@ -1 +1 @@
-sbt.version=0.13.16
+sbt.version=1.0.3
```

Comment:
There was no any problem updating and I was doubting if it was kept on 0.13 for some reason

Reply:
We started with sbt 1, but the file watcher is broken (at least on macos), so I would stick to 0.13 until it's fixed.

Link to the issues: https://github.com/sbt/sbt/issues/3687

and upstream https://github.com/sbt/io/issues/82

Reply:
Yes, file watching was a problem for me with sbt 1.0

Reply:
I think there are some fixes, more info in the upstream ticket.

Reply:
You can revert file watching with https://github.com/sbt/sbt/pull/3626.

Reply:
So as far I understood, despite the workaround for the filewatcher we're staying on sbt 0.13

Reply:
Done in e7a7ec3